### DISCUSSION TITLE
No operation, but the CPU is very high

### DISCUSSION THREAD

**Discussion by ljwps** [apache/doris#14416] [A - General / Q&A]
No queries and imports

doris 0.15 , 1fe + 3be
![image](https://user-images.githubusercontent.com/18318466/202839287-ea65a442-3d47-48f3-b0d9-a09c3e6c053e.png)

![image](https://user-images.githubusercontent.com/18318466/202839278-aadb12af-1441-45ef-9d00-483842dc678f.png)

**Comment by LemonLiTree** [comment]
This is a normal compaction task.
